CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 ## CHANGE LOG
 
+### Version 2.2.0 *(19 March 2024)*
+-------------------------------------------
+**What's new**
+* **[Android Platform]**
+  * Supports [CleverTap Android SDK v6.1.1](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md#version-611-february-27-2024).
+
+* **[iOS Platform]**
+  * Supports [CleverTap iOS SDK v6.1.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/6.1.0).
+  * Adds privacy manifests.
+
+* **[Android and iOS Platform]**
```

Comment:
These 2 points are not required in flutter docs as we never released a version supporting v6.1.0